src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__link {
+    padding-top: 10px;
+  }
+
+  @media (max-width: 1024px) {
+    width: 944px;
+  }
+}
```

Comment:
I think the header should have full-screen width and work correctly on 1024px resolution. But it hasn't been with 944px width